test/integration/esm-cjs.ts

```diff
@@ -406,43 +409,43 @@ type PluginConfig = {
   }
 
   const cjsTests = async () => {
-    await test('Install CJS plugin to CJS root plugin', async () => {
-      await installTest(PLUGINS.cjs2, cjsExecutor)
-    })
+    // await test('Install CJS plugin to CJS root plugin', async () => {
```

Comment:
why are these commented out?

Reply:
Commented out for iterating on tests - thanks for catching!